src/main/resources/data/c/oregen/auto_enable.json5

```diff
@@ -0,0 +1,11 @@
+{
+	//"ores": [
+		/*
+		"copper", "silver", "lead", "zinc", "aluminum", "cobalt", "tin", "titanium", "tungsten",
+		"platinum", "palladium", "osmium", "iridium",
+		"coal_coke",
+		"uranium",
+		"ruby", "topaz", "amethyst", "peridot", "sapphire"
+		*/
+	//]
```

Comment:
That’s a template file?

Reply:
Yeah, it's meant to enable everything if uncommented. I still have some things to add.